EIPS/eip-7923.md

```diff
@@ -30,7 +30,7 @@ There are two factors which contribute to "cold" memory (i.e. not-recently-used)
 
 This EIP uses a virtual addressing scheme so that memory pages are not allocated until they are actually accessed. Further, it adds a surcharge for accessing memory outside of an EVM-defined "hot" area.
 
-Notably, the data structures used for costing memory do not need to be part of the memory implementation itself, which suggests an elegant implementation using `mmap`.
+Notably, the data structures used for costing memory do not need to be part of the memory implementation itself, which suggests an elegant implementation using the POSIX `mmap` syscall (or, its counterpart on Windows, `VirtualAlloc`).
```

Comment:
:+1: for also including the Windows variant :smile: This makes it much easier to understand the idea of this underlying `mmap` for non-linux users.